composer.json

```diff
@@ -31,5 +35,10 @@
     "psr-0": {
       "Test\\PrometheusPushGateway\\": "tests/"
     }
+  },
+  "suggest": {
```

Comment:
I guess suggesting guzzle should also be part here or?

Reply:
@LKaemmerling guzzle is covered by https://packagist.org/providers/psr/http-client-implementation which is a virtual package that help list all PSR Http client supporting libraries. As you can see, Guzzle is first in the result 😉